From b5ddfcda321dcf5c65f71fec560642f2a48a3aa5 Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Thu, 15 Nov 2012 11:18:39 +0800
Subject: [PATCH 369/486] ENGR00233400 compile: fix compile error

wifi module should compile everytime. But From the Makefile,
ifneq ($(KERNELRELEASE),) it won't compile. refactor the compile
branch to fix sometimes it will leave without compile which will
cause $ACP error.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 wlan/atheros/compat-wireless/Makefile | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/wlan/atheros/compat-wireless/Makefile b/wlan/atheros/compat-wireless/Makefile
index 8c571ca..23c1582 100644
--- a/wlan/atheros/compat-wireless/Makefile
+++ b/wlan/atheros/compat-wireless/Makefile
@@ -20,8 +20,6 @@ ccflags-y += \
         -DCOMPAT_PROJECT="\"Compat-wireless\"" \
         -DCOMPAT_VERSION="\"$(shell cat $(COMPAT_ROOT)/compat_version)\""
 
-ifneq ($(KERNELRELEASE),)
-
 NOSTDINC_FLAGS := -I$(M)/include/ \
 	-include $(M)/include/linux/compat-2.6.h \
 	$(ccflags-y)
@@ -33,19 +31,8 @@ obj-$(CONFIG_COMPAT_RFKILL) += net/rfkill/
 ifeq ($(BT),)
 obj-$(CONFIG_COMPAT_WIRELESS) += net/wireless/ net/mac80211/
 obj-$(CONFIG_COMPAT_WIRELESS_MODULES) += drivers/net/wireless/
-
-
-
-
-ifeq ($(CONFIG_STAGING_EXCLUDE_BUILD),)
-endif
-
 endif
 
-
-else
-
-
 # These exported as they are used by the scripts
 # to check config and compat autoconf
 export COMPAT_CONFIG=config.mk
@@ -455,7 +442,5 @@ wlunload:
 
 .PHONY: all clean install uninstall unload btunload wlunload modules bt
 
-endif
-
 clean-files += Module.symvers Module.markers modules modules.order
 clean-files += $(CREL_CHECK) $(CONFIG_CHECK)
-- 
1.8.0

